docs/flink/flink-getting-started.md

```diff
@@ -302,7 +302,7 @@ CREATE TABLE `hive_catalog`.`default`.`sample` (
 
 Table create commands support the most commonly used [flink create clauses](https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/sql/create.html#create-table) now, including: 
```

Comment:
Maybe consider updating this link from 1.11 to 1.14 as well? https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/dev/table/sql/create/ is the new link.

Reply:
I have changed the flink version from 1.11 to 1.14，please review this change, thank you.